site/content/entry/citra-progress-report-2018-Q1.md

```diff
@@ -0,0 +1,245 @@
+enabled, it makes use of host GPU and when disabled, it uses CPU.
+
+Now as many of the users were using older hardware, we introduced an option called
+**Accurate Hardware Shader**. It was later renamed to **Accurate Multiplication**,
+because technically that's what it was doing. ***When enabled it would render games
+with more accuracy at the cost of slower performance***. Its the same with
+**Accurate Geometry Shader**. Earlier it was configurable via the ini file, but
+now its a check box to enable or disable accurate geometry shader.
+
+&nbsp;
+{{< sidebyside "image" "/images/entry/citra-progress-report-2018-q1/"
+    "renderer_settings.png=Old Graphics Tab"
+    "renderer_settings_new.png=New Graphics Tab" >}}
+
+During its time in Canary, a lot of testing was done by our contributors.
+They tested many number of games, under varied hardware conditions and submitted
+us with valuable telemetry data, crash logs, etc., which helped us in fixing
+issues with the hardware shader support.
+
+**Note**:
+Unfortunately, due to driver issues with AMD GPUs and inadequate OpenGL support on
+macOS,the hardware shader might not perform better than the CPU renderer under
+these conditions. However, worry not, as research is being done into possible fixes
+for these and we might have them sooner than you think.
+
+_Read more about hardware renderer improvements in detail in [**here**](https://citra-emu.org/entry/improvements-to-hardware-renderer/)._
+
+## Continuous Integration (CI) Improvements ([here](https://github.com/citra-emu/citra/pull/3647), [here](https://github.com/citra-emu/citra/pull/3649), [here](https://github.com/citra-emu/citra/pull/3706) and [here](https://github.com/citra-emu/citra/pull/374)) by [liushuyu](https://github.com/liushuyu)
+
+One of our major time and resource consuming parts of the project were our build
+generating environments. We use Travis and Appveyor to release fully functional
+builds to be downloaded by users. However, the time it took for a build to be
+generated was very high and if we stopped the building process abruptly, to do a
+quick fix, it took even more time to generate the build again.
+Thanks to [liushuyu](https://github.com/liushuyu)'s contributions, we were able
+to optimize this whole process and make it very efficient to easily build and
+release latest Citra Nightly and Canary releases.
+
+## [Logging Rewrite](https://github.com/citra-emu/citra/pull/3533) by [jroweboy](https://github.com/jroweboy) and [daniellimws](https://github.com/daniellimws)
+
+Our old logging system was becoming obsolete and a newer and efficient logging
+system based on fmtlib was designed and implemented by [jroweboy](https://github.com/jroweboy)
+and [daniellimws](https://github.com/daniellimws).
+It got rid of the ugly console window and made many under the hood improvements
+to the logging system.
+With this, identifying errors from crash logs became a whole lot easier and it
```

Comment:
I believe this is faulty parallelism. You can't say "became a.." and then "will help us", instead I would structure it like "this will make identifying errors and fixing issues faster."